pages/members/[id].js

```diff
@@ -8,16 +8,18 @@ export default function MemberDetails() {
   if (!member) return null;
   return (
     <div>
-      <h1>Details of members</h1>
-      <ul>
-        <li>
+      <h1 className="text-secondary-300 text-6xl text-center px-16 py-16">
+        Details of members
+      </h1>
+      <ul className="mt-2 mx-80 my-80 mb-10 text-lg">
+        <li className="px-16 py-16 text-center border-b border-gray-300 text-3xl text-bold text-gray-600 pb-10">
           <p>{member.name}</p>
         </li>
-        <li>
+        <li className="px-16 py-16 text-center text-base pt-8">
           <p>{member.email}</p>
         </li>
       </ul>
-      <p>
+      <p className="flex justify-center h-80 width-full object-cover mb-40 mt-10">
```

Comment:
i wont get into details of semantic html but this is not a paragraph (p). I just want to point out that accessibility and semantic html are points of consideration in web development.

Reply:
then what should I do with that?

Reply:
you can just use div here for example. this is a broad topic which we cant cover over night.